recipe/meta.yaml

```diff
@@ -12,14 +12,15 @@ source:
   patches:
     # fixes linker error on linux platforms, see https://github.com/pytorch/pytorch/issues/90448
     - patches/fix_kDefaultTimeout.patch
+    - patches/0001-windows-FindMKL-add-library-suffix.patch  # [win]
 
 build:
-  number: 0
+  number: 1
   skip: True  # [py<38]
   # Dropping ppc because of various build issues
   skip: True  # [(linux and ppc64le)]
-  # Only building for linux-amd64 GPU in this PR
-  skip: True  # [not (linux and x86_64)]
+  # Only building for win-64 CPU in this PR
+  skip: True  # [not win]
```

Comment:
I suppose this is a temporary change to avoid spending time on other builds, we need to get generic setting, if those are supposed to be adhoc, then consider introducing variants ?